docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,180 @@
+  For example, content can be computed, edited or served in specific ways such as
+  scheduled/live imports/exports, automated edition tasks, or specific controllers to communicate with other applications.
+
+![Ibexa Headless data inputs and outputs](img/headless.png)
+
+## Capabilities and benefits
+
+[[= product_name_headless =]] is a tool box with a Back Office.
+It comes without a default front office.
+You don't lose time to develop a theme for a provided front office before discovering it won't fit your needs.
+No distraction.
+[[= product_name_headless =]] helps you focus on the content, create and organize with its straightforward user interface (UI),
+imagine its inputs/outputs, and implement them with its various layers' APIs.
+
+### Core features
+
+The core of [[= product_name =]] Headless edition is already awesome as it offers everything to structure your content repositories and access them.
+
+#### Content model
+
+- Content items are organized as a tree in a repository.
+- An item can have multiple locations in this tree.
+- Content items are typed.
+- Content types are sets of typed data fields, with eventually conditions on the possible values.
+- Rich Text Field Type comes with an online editor. See [Online Editor product guide](online_editor_guide.md) for more.
+- Multilingual, it can store a content in several languages, the content model define which field must be translated, and which don't vary.
+
+See [Content management product guide](content_management_guide.md) for more.
+
+#### User management
+
+- User and user group rights are set by roles with thin granular limited permission policies in a safe deny-by-default security system.
+- Users are content items as well, so your knowledge about content management is reused.
+
+See [User management product guide](user_management_guide.md) for more.
+
+#### Content access
+
+- The REST API and GraphQL API support access to and edition of the content.
+- [[= product_name_headless =]] offers a complete PHP API to extend the ways to access content.
+- A design engine and a view controller offer to create plain text content views (such as HTML, JSON, XML, CSS, JS, CSV, or Markdown),
+  and to factorize those views by using theme cascades.
+  This design engine is used in the Back Office which is equally extendable.
+- Multichannel, content can be accessed through several channel configurations,
+  such as the domain name it replies to, the sub-part of the content tree it starts from, the users rights, or the design theme.
+  The Back Office itself is such a channel.
+- Multi-repository, the same platform can use separate databases if impermeability is needed between channel groups.
+
+### Advanced features
+
+On top of this strong core, [[= product_name =]] Headless edition brings tools to increase user experience, from final front users to Back Office contributors.
+
+#### [[= product_name_headless =]] is a complete Digital Experience Platform (DXP)
```

Comment:
I would  change the header of the section to "Complete DXP" and reuse this statement by expanding it into an introductory sentence...

"[[= product_name_headless =]] is a complete Digital Experience Platform (DXP), which comes with the following components to help users with content delivery:"

Reply:
```suggestion
#### Complete DXP

[[= product_name_headless =]] is a complete Digital Experience Platform (DXP), which comes with the following components to enhance user's journey:
```